test/nodetool/test_nodetool.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import pytest
+import rest_api_mock
+import json
+import subprocess
+
+
+def test_jmx_compatibility_args(server_context, nodetool):
```

Comment:
Why is this called "test_jmx_compatibility_args"? It looks like it's testing "nodetool compact", how is it related to JMX?

Reply:
This is testing that parameters only used to regulate the connection to JMX by the C* nodetool are correctly ignored. I cannot just call `nodetool`, so I had to use a command too.